src/coreclr/vm/interoputil.cpp

```diff
@@ -2228,14 +2241,24 @@ OLE_COLOR ConvertSystemColorToOleColor(OBJECTREF *pSrcObj)
     }
     CONTRACTL_END;
 
-    // Retrieve the method desc to use for the current AD.
-    MethodDesc *pSystemColorToOleColorMD =
-        GetAppDomain()->GetLoaderAllocator()->GetMarshalingData()->GetOleColorMarshalingInfo()->GetSystemColorToOleColorMD();
-    MethodDescCallSite systemColorToOleColor(pSystemColorToOleColorMD);
+    OLE_COLOR result;
+    OBJECTREF sysColor = nullptr;
```

Comment:
```suggestion
    OBJECTREF sysColor = NULL;
```

My guess is the rest of this file uses `NULL`. If not, feel free to ignore.